packages/accounts/api/user.ts

```diff
@@ -18,10 +18,15 @@ import { APPS, OTT_CLIENTS } from '@ente/shared/apps/constants';
 
 const ENDPOINT = getEndpoint();
 
-export const sendOtt = (appName: APPS, email: string) => {
+export const sendOtt = (appName: APPS, email: string, referral?: string) => {
+    let cleanedReferral = referral.trim();
+    if (!cleanedReferral?.length) {
+        cleanedReferral = undefined;
+    }
     return HTTPService.post(`${ENDPOINT}/users/ott`, {
         email,
         client: OTT_CLIENTS.get(appName),
+        referral: cleanedReferral,
```

Comment:
We need to send this information during verifyOTT because during verifyOtt we create the user if needed.

Reply:
oops, fixing